tests/unit/test_openapi/test_request_body.py

```diff
@@ -217,3 +217,69 @@ async def handler(
             }
         }
     }
+
```

Comment:
I think these tests would be more readable if you used `pytest.mark.parametrize`